situp-core/src/test/java/com/amazon/situp/TestDataProvider.java

```diff
@@ -1,253 +1,45 @@
 package com.amazon.situp;
 
-import com.amazon.situp.model.configuration.Configuration;
 import com.amazon.situp.model.configuration.PluginSetting;
-import com.amazon.situp.parser.model.PipelineConfiguration;
 
-import java.io.BufferedReader;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
-import java.util.Objects;
 import java.util.Set;
 
 public class TestDataProvider {
-    public static String TEST_PLUGIN_NAME = "test-plugin";
-    public static final String VALID_CONFIGURATION_FILE = "valid_pipeline_configuration.yml";
+    public static final String TEST_PLUGIN_NAME_1 = "test-plugin-1";
+    public static final String TEST_PLUGIN_NAME_2 = "test-plugin-2";
+    public static final int TEST_WORKERS = 5;
+    public static final int TEST_DELAY = 3_000;
```

Comment:
We should check the bounds for the Workers and Delay?

Reply:
good point - i thought we decided to work on bounds during bench mark testing; We can add really basic bound checks and error out like negative or 0, currently it defaults to default value when it finds 0 or negatives